packages/calcite-components/src/components/combobox/combobox.tsx

```diff
@@ -531,6 +532,9 @@ export class Combobox
 
   @State() text = "";
 
+  /** keeps track of the tooltipText */
+  @State() tooltipText: string;
```

Comment:
maybe this should just change to `@State() showTooltipText = false;`

No need to store the string value since its just the selected item's display value.

Reply:
Agreed.  Storing the value is duplicative.  You should be able to get the selected item's value with the element reference in the ResizeObserver callback function.